docs/releasing/publishing-from-a-support-branch.md

```diff
@@ -0,0 +1,162 @@
+# Publish GOV.UK Frontend when `main` branch has unreleasable changes
+
+Follow this document to publish a support branch of GOV.UK Frontend when the `main` branch contains unreleasable changes. For example, you might need to release a fix as part of a:
+
+- patch release, after the team has started to merge changes for a new feature release into the `main` branch - for example, a 3.14.x release once we've started merging changes for 3.15.0
+- release, after the team has started to merge changes for a new breaking release into the `main` branch - for example, a 3x.x release once we've started merging changes for 4.0.0
+
+Do not use this document to release changes for previous major versions. We have not tested the document in this scenario, and extra work would be needed to tell NPM not to mark the release as the 'latest'.
+
+If the `main` branch contains unreleased changes that are releasable immediately, [follow the steps in Publishing GOV.UK Frontend](/docs/releasing/publishing.md) instead.
+
+If the `main` branch only has a few unreleasable changes, you can temporarily revert these changes.
+
+1. Revert the unreleasable changes on the `main` branch.
+2. Publish GOV.UK Frontend.
+3. Add the reverted changes back into the `main` branch.
+
+However, this approach has risks. For example, it creates a messy commit history on the `main` branch.
+
+## Release GOV.UK Frontend from support branch
+
+### Before you release GOV.UK Frontend
+
+1. At stand up, tell the GOV.UK Design System team we are close to releasing.
+
+2. Developers to raise new issues in the team GitHub repositories ([govuk-frontend](https://github.com/alphagov/govuk-frontend), [govuk-frontend-docs](https://github.com/alphagov/govuk-frontend-docs), [govuk-prototype-kit](https://github.com/alphagov/govuk-prototype-kit)) to:
+    - create announcement draft for the new release (example card: [#2108](https://github.com/alphagov/govuk-frontend/issues/2108))
+    - create release notes for the new release (example card: [#1986](https://github.com/alphagov/govuk-frontend/issues/1986))
+    - create release notes for the new version of GOV.UK Prototype Kit (example card: [#958](https://github.com/alphagov/govuk-prototype-kit/issues/958))
```

Comment:
@vanitabarrett In the bullets before this one, we mention 'releases'. Is there a difference between a 'release' and a 'version'?

Reply:
That's a good question! I think we use them a bit interchangeably at the moment. In this example, I think we could talk about the 'new release of the GOV.UK Prototype Kit' and that would make sense and be more consistent.

Reply:
Thanks @vanitabarrett!

@jonathanglassman I'll go through this doc and, where appropriate, replace 'version' with 'release'.

```suggestion
    - create release notes for the new release of GOV.UK Prototype Kit (example card: [#958](https://github.com/alphagov/govuk-prototype-kit/issues/958))
```

Reply:
thanks!